README.md

```diff
@@ -36,7 +36,14 @@ and discussion, and please direct specific questions to [Stack Overflow](https:/
 
 People who are a little more adventurous can also try our nightly binaries:
 
+**Nightly pip packages**
+* We are pleased to announce that TensorFlow now offers nightly pip packages
+under the [tf-nightly](https://pypi.python.org/pypi/tf-nightly) project on pypi.
+We currently only support CPU-only packages on Linux and Mac. GPU packages on all platforms
```

Comment:
Also mention 
"You can just run `pip install tf-nightly` to install nightly tensorflow  build."
Or something that sounds nicer :)